src/opaque.rs

```diff
@@ -539,6 +90,14 @@ impl<CS: CipherSuite> ClientRegistration<CS> {
     }
 }
 
+/// Optional parameters for client registration start
+pub enum ClientRegistrationStartParameters {
```

Comment:
nit: this could actually implement (and use) [`core::default::Default`](https://doc.rust-lang.org/core/default/trait.Default.html)